ajax/libs/tracing.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "tracing-js",
+	"filename": "tracing.min.js",
+	"description": "Tracing.js is a library/utility to help you debug and trace javascript code. Tracing.js allows you to set functions of your own to be called before or after target functions in case of debugging get called. It is like events for function invocations.",
+	"license": "MIT",
+	"author": "ebobby",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/ebobby/tracing.js"
+	},
+	"keywords": [
+		"tracing",
+		"debug",
+		"development",
+		"javascript"
+	],
+	"npmName": "tracing-js",
+	"npmFileMap": [{
+		"basePath": "dist",
+		"files": [
+			"**/!(*.gz)"
+		]
+	}],
+	"autoupdate": {
+		"source": "git",
+		"target": "git://github.com/ebobby/tracing.js",
+		"fileMap": [{
+			"basePath": "dist",
+			"files": [
+				"**/!(*.gz)"
```

Comment:
Let's use `tracing*` to fetch `tracing.js`. For more details about this pattern, please see https://github.com/isaacs/node-glob